modules/homebrew/init.zsh

```diff
@@ -39,14 +39,11 @@ alias brewu='brew upgrade'
 alias brewx='brew uninstall'
 
 # Homebrew Cask
-alias cask='brew cask'
-alias caskc='hb_deprecated brew cask cleanup'
-alias caskC='hb_deprecated brew cask cleanup'
-alias caski='brew cask install'
-alias caskl='brew cask list'
-alias casko='brew cask outdated'
-alias casks='hb_deprecated brew cask search'
-alias caskx='brew cask uninstall'
+alias caski='brew install --cask'
+alias caskl='brew list --cask'
+alias casko='brew outdated --cask'
+alias casks='brew search --cask'
+alias caskx='brew uninstall --cask'
 
 function hb_deprecated {
```

Comment:
Shouldn't this also be removed since it's no longer used??

Reply:
Well, kinda :) Actually, we shouldn't be abruptly dropping alias without a migration warning. This is in my backlog. Will cherry-pick the commit and apply the necessary changes.

Reply:
While I am not opposed to a migration warning, I am also _completely_ fine with dropping aliases without a migration warning. I _really_ don't want the expectation to be that we have to do a formal migration for everything, especially more minor things like simple aliases.

So far we've kept the focus that this is a low-maintenance thin shell wrapper targeting power users, so I think it's fine to have an assumption of "read the commit log when you upgrade" so as long as we clearly document backwards breaking changes it in the commit title, then I think that's sufficient. We aren't that active of a project, so as long as you update once or twice a year it should take < 10 mins to skim the commit log.

Again, I am totally not opposed if you want to go to the additional work of adding migration warnings, as long as it doesn't become an expectation/requirement except for core functionality changes (and even then, I'm often still fine with "the migration message is the commit history").

Reply:
Okay @jeffwidman, compelling arguments! 👍🏽